src/main/java/Event.java

```diff
@@ -1,16 +1,20 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
 public class Event extends Task {
 
-    protected String from;
-    protected String to;
+    protected final LocalDate from;
+    protected final LocalDate to;
 
-    public Event(String description, String from, String to) {
+    public Event(String description, LocalDate from, LocalDate to) {
         super(description);
         this.from = from;
         this.to = to;
     }
 
     @Override
```

Comment:
Could make use of inherit doc to inherit the overridden public method's java docs
i.e.
```
/**
 * {@inheritDoc}
 * /
```